.CI/build_all

```diff
@@ -19,7 +19,7 @@ conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
 # install conda-build 2.x to build with a long prefix
-conda install --yes --quiet conda-build=2
+conda install --yes --quiet conda-build=2.0.12
```

Comment:
Yeah, that's happening when running `source run_conda_forge_build_setup`. This line is after that so it shouldn't matter what `conda-forge-build-setup` does.